.github/workflows/test_check_docstrings.yaml

```diff
@@ -0,0 +1,17 @@
+name: Test Check Docstrings
+on:
+  workflow_dispatch:
+
+jobs:
+  check-docstrings:
+    runs-on: ubuntu-latest
+    name: Check Docstrings
+    steps:
+      - id: check_docstrings
+        uses: catalystneuro/.github/.github/actions/check_docstrings@main
```

Comment:
I think it would be good to have this workflow be called with a relative path so you can use it to test the current commit instead of the version in main:

https://docs.github.com/en/actions/sharing-automations/reusing-workflows#calling-a-reusable-workflow